zanata-war/src/main/java/org/zanata/limits/RateLimitManager.java

```diff
@@ -125,7 +125,7 @@ public String getFieldValueAsString(String fieldName) {
                 });
     }
 
-    public RestCallLimiter getLimiter(final String apiKey) {
+    public RestCallLimiter getLimiter(final String key) {
```

Comment:
Oh, it's because it could be an api key, or it could be an ip address?  Lumping them both into a String could be a bit error-prone.  Perhaps `Either` could help? http://stackoverflow.com/a/16579332/14379  And the JavaDoc should say what's going on.